src/actions/index.js

```diff
@@ -7,4 +7,6 @@
 
 export const dashboardUpdate = data => ({ type: "dashboard_update", payload: data });
 
-export const isVersionAllowedUpdate = data => ({ type: "is_version_allowed_update", payload: data });
\ No newline at end of file
+export const isVersionAllowedUpdate = data => ({ type: "is_version_allowed_update", payload: data });
+
+export const apiLoadErrorUpdate = data => ({ type: 'api_load_error_update', payload: data })
```

Comment:
Missing `;` in end of the line. (Maybe we should create an issue to add a linter in the near future)